### ISSUE TITLE
Call to undefined method daita\\MySmallPhpTools\\Model\\Request::setBaseUrl

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/circles#440]
```json
{
    "Exception": "Error",
    "Message": "Call to undefined method daita\\MySmallPhpTools\\Model\\Request::setBaseUrl()",
    "Code": 0,
    "Trace": [
      {
        "file": "…/apps/circles/lib/Service/GSUpstreamService.php",
        "line": 143,
        "function": "asyncBroadcast",
        "class": "OCA\\Circles\\Service\\GlobalScaleService",
        "type": "->",
        "args": [
          {
            "__class__": "OCA\\Circles\\Model\\GlobalScale\\GSEvent"
          }
        ]
      },
      {
        "file": "…/apps/circles/lib/Service/MembersService.php",
        "line": 607,
        "function": "newEvent",
        "class": "OCA\\Circles\\Service\\GSUpstreamService",
        "type": "->",
        "args": [
          {
            "__class__": "OCA\\Circles\\Model\\GlobalScale\\GSEvent"
          }
        ]
      },
```

**Comment by nickvergessen** [comment]
Seems like the dependency in composer was not fixed to a version and the function is not there anymore

**Comment by joshtrichards** [comment]
Fixed in #950
